### PR TITLE
add win C TTS SDK

### PR DESCRIPTION
**Pull request by neoguo0601**
Doesn’t support streaming yet. And doesn't support play, just download stream'

### CODE REVIEW

File: TTS_SDK/WIN_C/TTS_SDK_sample/TTS_client_Test.c [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#include<stdio.h>
+#include"iconv\iconv.h"
+#include"TTS_SDK.h"
+#include<time.h>
+#include"azure_c_shared_utility\httpheaders.h"
+#include"azure_c_shared_utility\httpapi.h"
+
+const unsigned char* ApiKey = "f2c0453f2e0e448584f76e3d25989df1";
+
+const char* text = "����˵���������ź�����û�г���һ��Ů��\
```
Comment (szhaomsft) [inline TTS_SDK/WIN_C/TTS_SDK_sample/TTS_client_Test.c]: pls use English sample

File: TTS_SDK/WIN_C/TTS_SDK_sample/TTS_client_Test.c [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+void main() {
+	MSTTS_HANDLE MSTTShandle = MSTTS_init(ApiKey); 
+	if (MSTTShandle == NULL) {
+		printf("init MSTTS failed");
+		return 0;
+	}
+
+	unsigned char* bodyText = malloc(MAX_BodyLength);
```
Comment (szhaomsft) [inline TTS_SDK/WIN_C/TTS_SDK_sample/TTS_client_Test.c]: I don't like hard coded buffer len

File: TTS_SDK/WIN_C/TTS_SDK_sample/TTS_client_Test.c [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+	if (bodyText == NULL) {
+		printf("malloc failed");
+		return 0;
+	}
+	memset(bodyText, 0, MAX_BodyLength);
+	conversion(text, bodyText, MAX_BodyLength, "utf-8", "GBK");
```
Comment (szhaomsft) [inline TTS_SDK/WIN_C/TTS_SDK_sample/TTS_client_Test.c]: have one single exit for C function

File: TTS_SDK/WIN_C/TTS_SDK_sample/TTS_client_Test.c [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+	unsigned char* wavStream = malloc(102400);
+	if (wavStream == NULL) {
+		printf("malloc failed");
+		return 0;
+	}
+	memset(wavStream, 0 , 10240000);
+	MSTTS_Synthesizer(MSTTShandle, bodyText, MSTTSContentType_PlainText, wavStream, 102400);
```
Comment (szhaomsft) [inline TTS_SDK/WIN_C/TTS_SDK_sample/TTS_client_Test.c]: why do we need a fixed buffer

File: TTS_SDK/WIN_C/TTS_SDK_sample/azure_c_shared_utility/httpapi.h [hunk 5]
```diff
@@ -0,0 +1,195 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```
Comment (szhaomsft) [inline TTS_SDK/WIN_C/TTS_SDK_sample/azure_c_shared_utility/httpapi.h]: where are these changes from?

File: TTS_SDK/WIN_C/TTS_SDK_sample/TTS_SDK.h [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+
+	typedef enum MSTTSContentType
+	{
+		MSTTSContentType_PlainText,
+		MSTTSContentType_SSML
+	}MSTTSContent;
+
+	typedef enum MSTTSFormatType
+	{
+		ssml_16khz_16bit_mono_tts,
```
Comment (szhaomsft) [inline TTS_SDK/WIN_C/TTS_SDK_sample/TTS_SDK.h]: we don't need to expose so many format to caller

they just get PCM from SDK in a streaming way

File: TTS_SDK/WIN_C/TTS_SDK_sample/TTS_SDK.h [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+
+	typedef enum MSTTSGenderType
+	{
+		Male,
+		Famale
+	}MSTTSGender;
+
+	typedef struct MSTTS_VOICE_HANDLE_TAG
+	{
+		MSTTSGender gender;
```
Comment (szhaomsft) [inline TTS_SDK/WIN_C/TTS_SDK_sample/TTS_SDK.h]: no need to have gender
just lang and voice name

File: TTS_SDK/WIN_C/TTS_SDK_sample/TTS_SDK.h [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+	*   MSTTShandle: The handle of the synthesizer instance.
+	*   pszContent: The content to speak, can be plain text or ssml depending to nContentType.
+	*	eContentType: 0: plain text, 1: SSML.
+	*	wavStream: the syntheticed data stream.
+	*	wavStreamLength: the length of the data stream 
+	* Return value:
+	*	MSTTS_OK if initialization is successful or an error
+	* 		code in case it fails.
+	*/
+	MSTTS_RESULT MSTTS_Synthesizer(MSTTS_HANDLE MSTTShandle, const char* pszContent, MSTTSContent eContentType, unsigned char* wavStream, size_t wavStreamLength);
```
Comment (szhaomsft) [inline TTS_SDK/WIN_C/TTS_SDK_sample/TTS_SDK.h]: why use fixed length 
you should implement a callback

File: TTS_SDK/WIN_C/TTS_SDK_sample/TTS_SDK.h (hunk 7 above)
Comment (neoguo0601) [inline TTS_SDK/WIN_C/TTS_SDK_sample/TTS_SDK.h]: OK, get it

File: TTS_SDK/WIN_C/TTS_SDK_sample/TTS_SDK.h (hunk 8 above)
Comment (neoguo0601) [inline TTS_SDK/WIN_C/TTS_SDK_sample/TTS_SDK.h]: When the support streaming, I will increase the callback

File: TTS_SDK/WIN_C/TTS_SDK_sample/TTS_SDK.h (hunk 6 above)
Comment (neoguo0601) [inline TTS_SDK/WIN_C/TTS_SDK_sample/TTS_SDK.h]: OK

File: TTS_SDK/WIN_C/TTS_SDK_sample/azure_c_shared_utility/httpapi.h (hunk 5 above)
Comment (neoguo0601) [inline TTS_SDK/WIN_C/TTS_SDK_sample/azure_c_shared_utility/httpapi.h]: Has not yet fixed support for Stream

File: TTS_SDK/WIN_C/TTS_SDK_sample/TTS_client_Test.c (hunk 4 above)
Comment (neoguo0601) [inline TTS_SDK/WIN_C/TTS_SDK_sample/TTS_client_Test.c]: Just test in sample, Iconv is not in the SDK, just test whether to support Chinese

**Pull request closed without merging**